test/server/server.py

```diff
@@ -27,3 +38,9 @@ async def read_item():
 @app.post("/503", status_code=503)
 async def read_item():
     return 503
+
+
+@app.get("/200/api_key")
```

Comment:
could you add another endpoint with a different API key so we can add a negative test case as well? That would be awesome!

Reply:
added a new container with wrong credentials/secret and used the same endpoint